client-app/shared/account/components/reorder-info.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <VcPopup :title="$t('shared.account.reorder_info_popup.title')">
+  <VcPopup :title="$t('shared.account.reorder_info_popup.title')" :is-mobile-fullscreen="true">
```

Comment:
using string `is-mobile-fullscreen` is equivalent to `:is-mobile-fullscreen="true"`